R/init.R

```diff
@@ -153,6 +153,23 @@ init <- function(data,
   # log
   teal.logger::log_system_info()
 
+  # invoke lockfile creation
+  # If user has setup the file, there is no need to compute a new one.
```

Comment:
I honestly think that no-one will ever use this feature and this unnecessarily overcomplicate implementation as well as documentation. For me, on a high level, this should remain as a "closed" functionality - i.e. user just take what it returns without much options to influence. 

YAGNI my friend :)

Reply:
@pawelru we use pre-computed .lockfiles for teal.gallery and we also use lockfile for study apps where we help upversion teal to the newest version. You can ask @vedhav if this is used or not :)

Reply:
I actually thought I could use it last week